app/command/other.js

```diff
@@ -1,21 +1,22 @@
 import cp from 'child_process'
 import { get as getContext } from '@ppzp/context'
+import vscode from 'vscode'
 
 import Feifei from '../webview/feifei.js'
 import noty from '../../lib/vscode-utils/noty/index.js'
 import { warn } from '../../lib/vscode-utils/prompt/confirm'
 
 export
 async function empty() {
-  if(await warn('确认清除所有数据吗？', '此操作将清除全部 ppz 数据', '确认清除'))
+  if(await warn(vscode.l10n.t('Clear all data. Are you sure?'), vscode.l10n.t('This operate will delete all PPZ data'), vscode.l10n.t('Confirm to delete')))
```

Comment:
- Are you sure to doing so?
- This will delete all PPZ data!
- Yes, delete the data